web/src/beta/lib/unsafeBuiltinPlugins/load.ts

```diff
@@ -0,0 +1,20 @@
+import { UnsafeBuiltinPlugin } from "./types";
+
+export const loadPlugins = async () => {
+  const urls = window.REEARTH_CONFIG?.unsafePluginUrls;
```

Comment:
So basically you are processing and loading unsafePluginUrls twice, once in the config and then again here, right?
I'm guessing that is a mistake. Only one should be necessary, and if possible I'd prefer it be in the config. 🙏🏻

Reply:
You are right. I fixed it:pray:
[fix review](https://github.com/reearth/reearth/pull/625/commits/f72e6dfd7a69e50dad6d4af24828b42a4f926f16)